azure-iot-device/samples/advanced-hub-scenarios/get_storage_x509.py

```diff
@@ -0,0 +1,164 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
```

Comment:
This file also appears that it does not use our library and should be removed (or updated to use it)